docker-compose.yml

```
@@ -0,0 +1,22 @@
+version: '3'
+services:
+  web-mpc:
+    container_name: web-mpc
+    restart: always
+    build: .
+    ports:
+      - "3000:8080"
```

probably want to key the same port
```suggestion
      - "8080:8080"
```